controller/model/session_handlers.go

```diff
@@ -41,6 +42,7 @@ func (handler *SessionHandler) newModelEntity() boltEntitySink {
 }
 
 func (handler *SessionHandler) Create(entity *Session) (string, error) {
+	entity.Id = cuid.New() //use cuids which are longer than shortids but are monotonic
```

Comment:
This is a swap @plorenz and I talked about. Monotonic ids for api sessions and sessions for 1) performance reasons and 2) so that strategies can see through cursors for lazy loading strategies.